### PR TITLE
Update hset.md

### PR DESCRIPTION
**Pull request by massoudmaboudi**
Update the description to match by adding multiple keys description with respective examples

### CODE REVIEW

File: commands/hset.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-Sets `field` in the hash stored at `key` to `value`.
-If `key` does not exist, a new key holding a hash is created.
-If `field` already exists in the hash, it is overwritten.
+Sets the specified fields to their respective values in the hash stored at `key`.
+
+This command overwrites any specified fields already existing in the hash. If `key` does not exist, a new key holding a hash is created.
```
Comment (itamarhaber) [inline commands/hset.md:3]:
```suggestion
This command overwrites any specified fields that already exist in the hash.
If `key` doesn't exist, a new key that stores the hash is created.
```

File: commands/hset.md [hunk 2]
```diff
@@ -11,4 +11,8 @@ If `field` already exists in the hash, it is overwritten.
 ```cli
 HSET myhash field1 "Hello"
 HGET myhash field1
+HMSET myhash field2 "Hi" field3 "World"
```
Comment (itamarhaber) [inline commands/hset.md:14]: `HMSET` is considered deprecated.

```suggestion
HSET myhash field2 "Hi" field3 "World"
```

File: commands/hset.md [hunk 3]
```diff
@@ -11,4 +11,8 @@ If `field` already exists in the hash, it is overwritten.
 ```cli
```
Comment (itamarhaber) [inline commands/hset.md:11]: I don't understand the added value of this change to the example.

**Review by itamarhaber — COMMENTED** [review]
Hello @massoudmaboudi and thanks for the suggestion.

File: commands/hset.md (hunk 2 above)
Comment (massoudmaboudi) [reply commands/hset.md:14]: Correct, you made it as I wanted to be.

File: commands/hset.md (hunk 3 above)
Comment (massoudmaboudi) [reply commands/hset.md:11]: `hset` can add multiple values. but the original explanation and example was showing only 1 field.
I wanted to demonstrate this command can add single and multiple fields without checking `hmset` command and its deprecation note.

**Comment by massoudmaboudi** [comment]
@itamarhaber 
Any update for this MR?

File: commands/hset.md (hunk 1 above)
Comment (itamarhaber) [inline commands/hset.md:3]:
```suggestion
This command overwrites the values of specified fields that exist in the hash.
If `key` doesn't exist, a new key holding a hash is created.
```

File: commands/hset.md (hunk 2 above)
Comment (itamarhaber) [inline commands/hset.md:14]:
```suggestion
HSET myhash field2 "Hi" field3 "World"
```

**Review by itamarhaber — APPROVED** [review]

**Pull request merged**